src/codeu/chat/common/User.java

```diff
@@ -52,11 +53,19 @@ public User read(InputStream in) throws IOException {
   public final String name;
   public final Time creation;
 
-  public User(Uuid id, String name, Time creation) {
+  /*
+   * The sentiment score is public since the only way to update it is by connecting to the natural
+   * language API, which only the server can do. As such, the client must request a new sentiment
+   * score from the server
+   */
+  public SentimentScore sentimentScore;
+
+  public User(Uuid id, String name, Time creation, SentimentScore score) {
 
     this.id = id;
     this.name = name;
     this.creation = creation;
+    this.sentimentScore = score;
```

Comment:
Does it make sense for sentiment score to be a variable into the constructor?  I'm only seeing it being passed new SentimentScore() anyways.